javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+const intID = window.setInterval(function () {
+  const header = document.querySelector('h1');
+  if (header.textContent === '4') {
+    header.textContent = '3';
+  } else if (header.textContent === '3') {
+    header.textContent = '2';
+  } else if (header.textContent === '2') {
+    header.textContent = '1';
+  } else if (header.textContent === '1') {
+    header.textContent = '~Earth Beeeelooowww Us~';
+  } else if (header.textContent === '~Earth Beeeelooowww Us~') {
+    window.clearInterval(intID);
+  }
+}, 2.0 * 1000);
```

Comment:
Do you normally count down at half speed?

Reply:
So what if I do? LOL. but i'll change it. it's more for mental pacing